contrib/delta-kernel-rs-cmake/CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ file(COPY ${ClickHouse_SOURCE_DIR}/contrib/openssl/include/ DESTINATION ${DELTA_
 set(DELTA_KERNEL_TARGET_PATH ${DELTA_KERNEL_RS_BINARY_DIR}/include)
 
 corrosion_set_env_vars(delta_kernel_ffi
+        "CARGO_HOME=${CMAKE_BINARY_DIR}/contrib/corrosion-cmake/"
```

Comment:
I guess we need this for all other rust libraries/executables as well? It should not hurt

Reply:
I'd avoid adding it just because it should not hurt.

Reply:
What I meant is that other places can have similar calls to `cbindgen` (or what it was?) that calls `cargo` w/o arguments and may go to the internet (correct me if am I wrong here)

Reply:
Yep, got it. But we don't have other places at the moment :slightly_smiling_face: